pkg/auth/util/provider_util_test.go

```diff
@@ -0,0 +1,46 @@
+package util_test
+
+import (
+	"github.com/rancher/rancher/pkg/auth/util"
+	"testing"
+)
+
+func TestEscapeUUID(t *testing.T) {
```

Comment:
Nit: it's often a good idea to start a test with `t.Parallel()`, if it doesn't rely on global state.